beacon-chain/operations/slashings/service.go

```diff
@@ -75,7 +79,9 @@ func (p *Pool) PendingProposerSlashings(ctx context.Context, state *beaconstate.
 	// Update prom metric.
 	numPendingProposerSlashings.Set(float64(len(p.pendingProposerSlashing)))
 
-	pending := make([]*ethpb.ProposerSlashing, 0, params.BeaconConfig().MaxProposerSlashings)
+	// Allocate pending slice with a capacity of min(len(p.pendingProposerSlashing), maxProposerSlashings)
+	// since the array cannot exceed the max and is typically less than the max value.
+	pending := make([]*ethpb.ProposerSlashing, 0, mathutil.Min(uint64(len(p.pendingProposerSlashing)), params.BeaconConfig().MaxProposerSlashings))
```

Comment:
Same here. Max?

Reply:
Same comment. No need to allocate extra data when there is less than max slashings available. 

Is there anything I can make more clear in the comment above?

Reply:
Same comment. No need to allocate extra data when there is less than max slashings available. 

Is there anything I can make more clear in the comment above?

Reply:
Same comment. No need to allocate extra data when there is less than max slashings available. 

Is there anything I can make more clear in the comment above?

Reply:
This is good

Reply:
Sorry for spam, i was on mobile! lol